quodlibet/po/fr.po

```diff
@@ -1301,7 +1303,7 @@ msgid ""
 "Downloads covers linked to by the artwork_url tag.This works with the "
 "Soundcloud browser."
 msgstr ""
-"Télécharge la jaquette liée à l'étiquette artwork_url. Cela fonctionne "
+"Télécharge les jaquettes liée à l'étiquette artwork_url. Fonctionne "
```

Comment:
Other similarly formed messages (for example the one just below in the diff) use the infinitive form. It would be more consistent to use it here as well, or change all the other ones to the imperative form.